### ISSUE TITLE
Sublime Text 3 support

### ISSUE THREAD

**Issue by drnikko** [poritsky/fountain-sublime-text#7]
Any plans to port to ST3?

**Comment by quack1** [comment]
It would actually be great if someone could port it to Sublime Text 3 :)

The package does not show up on the Package Control manager. So I followed the steps mentioned in the "Manually Download" [section](https://github.com/poritsky/fountain-sublime-text#method-2-download) and
1. The package seems to be installed and recognized by ST3
2. BUT, I don't have a "Fountain" entry in the languages available.

Any help ?

**Comment by poritsky** [comment]
I can look into this, though I thought this had been dealt with some time ago. See here:

https://github.com/poritsky/fountain-sublime-text/pull/8

**Comment by poritsky** [comment]
I just tested installing and using the Fountain Package in Sublime Text 3 Build 3065. It worked as expected.

What OS are you using?

**Comment by quack1** [comment]
I'm using (x)Ubuntu 14.10 x64 with the latest Sublime Text 3 (I just reinstalled it, just to be sure).
I'm able to find Fountain in the Package Control, but I still don't have the syntax highlighting and everything else, even after restarting ST3

**Comment by mifuyne** [comment]
@quack1 your issue might be related to #6 

I've submitted a pull request to fix this, along with one other issue I've discovered. I hope that helps :smile: (almost two months later :frowning: )

**Comment by quack1** [comment]
@mifuyne Thank you for the heads up :) I will for for your PR to be merged and I will try it again :)

Ok, just saw the PR #9 was merged. I will try it now.

**Comment by quack1** [comment]
Nice, it just works well now :) Thank you very much @mifuyne !

**Comment by mifuyne** [comment]
No problem :grin: glad to be of help!

**Comment by luandro** [comment]
Shouldn't this be closed?

**Comment by quack1** [comment]
Yes, it can be closed since the Pull Request #9 that solve the problem on ST3 :)
